### DISCUSSION TITLE
No TA checked in during scheduled office hour

### DISCUSSION THREAD

**Discussion by FilbertZBW** [sandboxnu/office-hours#583] [Q&A]
Hi, I found two times that no staff checked in during scheduled OH. Could you clarify when there are shifts and notify us?

**Comment by willstenzel** [comment]
Hi @FilbertZBW, our app displays the scheduled office hours calendar, which is set up by the course coordinator. We cannot ensure that the staff will show up during their scheduled office hours so this issue is our of our control. 

@liustanley can you confirm?

**Reply by FilbertZBW** [reply]
It seems that now all worked well. There might be some emergency shift of TA for that case. Thanks!
